packages/app/src/debug/DebugPageDetails.vue

```diff
@@ -0,0 +1,72 @@
+<template>
+  <div
+    class="flex flex-col w-full px-24px justify-center items-center align-middle"
```

Comment:
Should this render anything if the status doesn't fall into one of the error conditions below. For example, `FAILED` or `RUNNING` status would just end up reserving blank space

Reply:
Updated this to render only for specific run statuses